packages/experimental/Divider/src/Divider.types.ts

```diff
@@ -14,56 +14,67 @@ export interface DividerProps {
   /**
    * If a text or icon is passed, this dictates where content appears in the divider: at the start, centered, or towards the end.
    * @default 'center'
+   * Note: This prop is not supported on mobile platforms(Android & iOS).
    */
   alignContent?: DividerAlignment;
   /**
    * If no color is passed, the divider and its content are colored using different theme tokens depending on the value of this prop.
    * @default 'default'
+   * Note: This prop is not supported on mobile platforms(Android & iOS).
    */
   appearance?: DividerAppearance;
   /**
    * Pass an icon source to render an icon as content in the divider. Mutually exclusive with passing text as a child.
+   * Note: This prop is not supported on mobile platforms(Android & iOS).
    */
   icon?: IconProps;
   /**
    * The size of the Divider inset - the margin before the start and after the end of the divider.
    * @default 0
+   * Note : For mobile platforms, the insetSize prop is only applied to start of the component.
    */
   insetSize?: DividerInsetSize;
   /**
    * Whether the divider is rendered as a horizontal line or a vertical line.
    * @default false
+   * Note: This prop is not supported on mobile platforms(Android & iOS).
    */
   vertical?: boolean;
 }
 
 export interface DividerTokens extends LayoutTokens {
   /**
    * The color of the content passed into the divider. This is overriden if the `color` prop is set.
+   * Note: This prop is not supported on mobile platforms(Android & iOS).
```

Comment:
I think all the properties in tokens do not need this note, tokens are customisation support out of the example Fluent variations. These properties are not props for the component, so a developer building a Divider would not reference them unless explicitly looking to style differently. This also aligns current guidance to document props and SPEC correctly, and to other components like - https://github.com/microsoft/fluentui-react-native/blob/main/packages/components/Checkbox/src/Checkbox.types.ts
 where the props

Reply:
Done.